src/renderer/components/status-bar/status-bar.tsx

```diff
@@ -19,7 +20,9 @@ interface Dependencies {
 }
 
 const NonInjectedStatusBar = observer(({ items }: Dependencies & StatusBarProps) => {
-  const { left, right } = items.get();
+  const { left: leftItems, right } = items.get();
+
+  const left = [AutoUpdateComponent, ...leftItems];
```

Comment:
This should be already part of the `items` dependency.

Reply:
You mean it should be inserted by `statusBarItemsInjectable`?

Reply:
yes

Reply:
ok. I guess if there were for a need for just the status bar items from extensions we could create another (base) injectable